src/Login/LoginForm.tsx

```diff
@@ -77,6 +78,29 @@ export const LoginForm: React.FC<LoginFormProps> = () => {
                 setLoginAlertMsg({ message: err.message, variant: 'danger' });
             }
         }
+
+        try {
+            const res = await AxiosRequest.get('/users/status');
```

Comment:
Why are we doing this on the login form?

At one point we talked about returning this in the login response but we decided to make a separate endpoint and use it as needed since things aren't ready to be reactive but that would keep it the most relevant